scripts/antlr4/Cypher.g4

```diff
@@ -478,17 +481,33 @@ oC_UpdatingClause
         | oC_Delete
         ;
 
-oC_ReadingClause
+oC_OptionalReturnReadingClause
+    : kU_InQueryTableFunctionCall;
+
+oC_MandatoryReturnReadingClause
     : oC_Match
         | oC_Unwind
-        | kU_InQueryCall
+        | kU_InQueryBuiltInCall
         | kU_LoadFrom
         ;
 
+oC_ReadingClause
+    : oC_OptionalReturnReadingClause
+        | oC_MandatoryReturnReadingClause
+        ;
+
 kU_LoadFrom
     :  LOAD ( SP WITH SP HEADERS SP? '(' SP? kU_ColumnDefinitions SP? ')' )? SP FROM SP kU_ScanSource (SP? kU_ParsingOptions)? (SP? oC_Where)? ;
 
 kU_InQueryCall
+    : kU_InQueryTableFunctionCall
+        | kU_InQueryBuiltInCall
+        ;
+
+kU_InQueryTableFunctionCall
+    : ( kU_ProjectGraph SP? )? CALL SP oC_TableFunctionCall (SP? oC_Where)? ;
+
+kU_InQueryBuiltInCall
     : ( kU_ProjectGraph SP? )? CALL SP oC_FunctionInvocation (SP? oC_Where)? ;
```

Comment:
Is `CALL COUNT(*)` or `CALL CAST` a thing or can I remove `kU_InQueryBuiltInCall` altogether?